exchanges/bitfinex/bitfinex_wrapper.go

```diff
@@ -649,13 +649,23 @@ func (b *Bitfinex) SubmitOrder(ctx context.Context, o *order.Submit) (*order.Sub
 	var orderID string
 	status := order.New
 	if b.Websocket.CanUseAuthenticatedWebsocketForWrapper() {
-		orderID, err = b.WsNewOrder(&WsNewOrderRequest{
+		symbolPrefix := "t"
+		if o.AssetType == asset.MarginFunding {
+			symbolPrefix = "f"
```

Comment:
Just a note: margin funding offers isn't currently implemented and requires a different endpoint https://docs.bitfinex.com/reference/ws-auth-input-offer-new

Reply:
Acknowledged. I'm not using it so this was just a driveby